packages/volto/news/6371.breaking

```diff
@@ -0,0 +1,2 @@
+Dropping support for Node.js 18 since Volto only supports the last two LTS versions @sneridagh
```

Comment:
```suggestion
Dropped support for Node.js 18, since Volto only supports the last two LTS versions. @sneridagh
```

Reply:
We should also have something in the upgrade guide, right? Like from earlier: https://github.com/plone/volto/blob/main/docs/source/upgrade-guide/index.md?plain=1#L481

Reply:
Yes, also at the top, immediately after the main heading. Good catch.

Reply:
Added in d104e206dd875d03ff6ad0739138bb74a338b7fa